tst/EnergyPlus/unit/OutputReportTabularAnnual.unit.cc

```diff
@@ -0,0 +1,163 @@
+	SetupOutputVariable( "Exterior Lights Electric Energy [J]", extLitUse, "Zone", "Sum", "Lite2", _, "Electricity", "Exterior Lights", "General" );
+	SetupOutputVariable( "Exterior Lights Electric Energy [J]", extLitUse, "Zone", "Sum", "Lite3", _, "Electricity", "Exterior Lights", "General" );
+	SetupOutputVariable( "Exterior Lights Electric Power [W]", extLitPow, "Zone", "Average", "Lite1" );
+	SetupOutputVariable( "Exterior Lights Electric Power [W]", extLitPow, "Zone", "Average", "Lite2" );
+	SetupOutputVariable( "Exterior Lights Electric Power [W]", extLitPow, "Zone", "Average", "Lite3" );
+
+	DataGlobals::DoWeathSim = true;
+
+	GetInputTabularAnnual(); // this also calls setupGathering
+	EXPECT_EQ( OutputReportTabularAnnual::annualTables.size(), 1 );
+
+	std::vector<AnnualTable>::iterator firstTable = OutputReportTabularAnnual::annualTables.begin();
+	std::vector<std::string> fieldSetParams = firstTable->inspectTableFieldSets( 0 );
+
+	EXPECT_EQ( fieldSetParams[0], "EXTERIOR LIGHTS ELECTRIC ENERGY" );
+	EXPECT_EQ( fieldSetParams[2], "J" ); // m_varUnits
+	EXPECT_EQ( fieldSetParams[4], "2" ); // m_typeOfVar
+	EXPECT_EQ( fieldSetParams[5], "3" ); // m_keyCount
+	EXPECT_EQ( fieldSetParams[6], "2" ); // m_varAvgSum
+	EXPECT_EQ( fieldSetParams[7], "1" ); // m_varStepType
+
+}
+
+TEST_F( EnergyPlusFixture, OutputReportTabularAnnual_GatherResults )
+{
+	std::string const idf_objects = delimited_string( {
+		"Version,8.3;",
+		"Output:Table:Annual,",
+		"Space Gains Annual Report, !- Name",
+		", !- Filter",
+		", !- Schedule Name",
+		"Exterior Lights Electric Energy, !- Variable or Meter 1 Name",
+		"SumOrAverage, !- Aggregation Type for Variable or Meter 1",
+		"4, !- field Digits After Decimal 1",
+		"Exterior Lights Electric Power, !- Variable or Meter 2 Name",
+		"Maximum, !- Aggregation Type for Variable or Meter 2",
+		", !- field Digits After Decimal 2",
+		"Zone Electric Equipment Total Heating Energy; !- Variable or Meter 3 Name",
+	} );
+
+	ASSERT_FALSE( process_idf( idf_objects ) );
+
+	Real64 extLitPow;
+	Real64 extLitUse;
+
+	SetupOutputVariable( "Exterior Lights Electric Energy [J]", extLitUse, "Zone", "Sum", "Lite1", _, "Electricity", "Exterior Lights", "General" );
+	SetupOutputVariable( "Exterior Lights Electric Energy [J]", extLitUse, "Zone", "Sum", "Lite2", _, "Electricity", "Exterior Lights", "General" );
+	SetupOutputVariable( "Exterior Lights Electric Energy [J]", extLitUse, "Zone", "Sum", "Lite3", _, "Electricity", "Exterior Lights", "General" );
+	SetupOutputVariable( "Exterior Lights Electric Power [W]", extLitPow, "Zone", "Average", "Lite1" );
+	SetupOutputVariable( "Exterior Lights Electric Power [W]", extLitPow, "Zone", "Average", "Lite2" );
+	SetupOutputVariable( "Exterior Lights Electric Power [W]", extLitPow, "Zone", "Average", "Lite3" );
+
+	DataGlobals::DoWeathSim = true;
+	DataGlobals::TimeStepZone = 0.25;
+
+	GetInputTabularAnnual();
+	EXPECT_EQ( OutputReportTabularAnnual::annualTables.size(), 1 );
```

Comment:
I have a suspicion that this is the line that is causing the build warning.  ```size()``` is probably returning a ```const unsigned int```, and 1 is being interpreted as just ```const int```.  @kbenne or @mbadams5 could you comment since this is only showing up as a build warning on Mac so I can't verify locally.